src/pages/NotificationsPage/components/notification-dropdown.tsx

```
@@ -0,0 +1,285 @@
+    <div className={cn("relative", className)} ref={dropdownRef}>
+      {/* Bell Icon Button */}
+      <button
+        onClick={() => setIsOpen(!isOpen)}
+        className={cn(
+          "relative rounded-full p-2 transition-colors",
+          isOpen
+            ? "bg-primary text-white hover:bg-primary"
+            : "text-primary hover:bg-primary-10",
+        )}
+        aria-label="Notifications"
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)